app/policies/project_policy.rb

```diff
@@ -21,7 +21,7 @@ class TranslateScope < Scope
         :create_subjects_export,
         :create_workflows_export,
         :create_workflow_contents_export,
-        :retire_subjects, with: WriteScope
+        :retire_subjects, :copy, with: WriteScope
```

Comment:
How will users copy template projects they don't collab / own? Should this be `ReadScope` i.e. public projects or ones i can see in my lab?

Reply:
Depends. Write makes sense to me to roll out this feature on a small number of hand selected projects (via the manually setting the configuration) and the button in PFE doesn't exist yet. For those same reasons, though, I can see not mattering very much and maybe it's better to put it in Read now and not have to change it later.